.devops/deploy-pipelines.yml

```diff
@@ -0,0 +1,134 @@
+# Azure DevOps pipeline to release a new version and publish the package to the registry
+#
+# The following variables must be configured in the pipeline settings
+#
+# GIT_EMAIL         - when bumping the package version, is the email we author the commit with
+# GIT_USERNAME      - when bumping the package version, is the name we author the commit with
+# GITHUB_CONNECTION - name of the Github service connection used to create a new release; be sure that the related user has appopriate right
+# NPM_CONNECTION    - name of the NPM service connection used to publish the package; be sure that the related user has appopriate right
+# 
+
+variables:
+  NODE_VERSION: '10.14.1'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+  gitReference: ${{ replace(replace(variables['Build.SourceBranch'], 'refs/tags/', ''), 'refs/heads/', '') }}
+  releaseTemplateName: templates/node-github-release/template.yaml@pagopaCommons
+  setupTemplateName: azure-templates/setup-project.yml
+
+parameters:
+  - name: 'RELEASE_SEMVER'
+    displayName: 'When packing a release, define the version bump to apply'
+    type: string
+    values:
+      - major
+      - minor
+      - patch
+    default: minor
+
+# Only manual activations are intended
+trigger: none
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
```

Comment:
when publishing packages we must use win32 build

Reply:
Actually we setup the same for all the packages, like [io-functions-commons](https://github.com/pagopa/io-functions-commons/blob/1af5e8cdd8cd53a329908be0cd8e037425dd24c0/.devops/deploy-pipelines.yml#L29) and [io-ts-commons](https://github.com/pagopa/io-ts-commons/blob/a81efc9983083688e2d9621a6eca99ec822ef9bb/.devops/deploy-pipelines.yml#L30)

What should I do, change all of them?

Reply:
nope, sorry I was wrong it's ok as is (moreover, this ib is used inside appbackend which is deployed on linux machines)

Reply:
good :)